siliconcompiler/schema/schema_obj.py

```diff
@@ -31,6 +32,40 @@ def __init__(self, cfg=None):
         else:
             self.cfg = copy.deepcopy(cfg)
 
+    ###########################################################################
+    @classmethod
+    def from_manifest(cls, filepath):
```

Comment:
Shouldn't this replace read_manifest with the new schema object approach?

Reply:
Not exactly the same, `read_manifest()` has additional logic for merging. Just added a comment in the main thread providing more detail.